platform/openide.awt/apichanges.xml

```diff
@@ -26,6 +26,20 @@
 <apidef name="awt">AWT API</apidef>
 </apidefs>
 <changes>
+    <change id="getArrowIcon">
+        <api name="awt"/>
+        <summary>Added factory class for drop-down buttons</summary>
```

Comment:
Forgot to change the summary (cut-and-pasted from another entry)... will change this one.

Reply:
Fixed this in a separate commit.